<ui version="4.0" >
 <author>Dag Andersen &lt;danders@get2net.dk></author>
 <class>KPlato::WorkPackageControlPanel</class>
 <widget class="QWidget" name="KPlato::WorkPackageControlPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>299</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Responsible:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="responsible" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="WorkPackageTableView" name="resourceTable" >
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="transferWP" >
       <property name="text" >
        <string>Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mailTo" >
       <property name="text" >
        <string>Mail To...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadWP" >
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewWP" >
       <property name="text" >
        <string comment="View Work Package">View...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkPackageTableView</class>
   <extends>QTableView</extends>
   <header>kptworkpackageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
